pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -576,7 +576,7 @@ func (dswp *desiredStateOfWorldPopulator) createVolumeSpec(
 func (dswp *desiredStateOfWorldPopulator) getPVCExtractPV(
 	namespace string, claimName string) (*v1.PersistentVolumeClaim, error) {
 	pvc, err :=
-		dswp.kubeClient.CoreV1().PersistentVolumeClaims(namespace).Get(claimName, metav1.GetOptions{})
+		dswp.kubeClient.CoreV1().PersistentVolumeClaims(namespace).Get(claimName, metav1.GetOptions{ResourceVersion: "0"})
```

Comment:
From a readability/maintainability standpoint, it concerns me a little that a change as simple as adding `ResourceVersion: "0"` is sufficient for the kubeClient to start consuming from the cache.

Could we make this impact clear via the code? For example, what about:

```go
optionsForcingApiServerCacheUsage := metav1.GetOptions{ResourceVersion: "0"}
dswp.kubeClient...Get(claimName, optionsForcingApiServerCacheUsage)
```

Let me know what you think about this idea :) Admittedly, I haven't spent a ton of time working with the `kubeClient` in this context, so it may be obvious to others that `ResourceVersion: "0"` has this impact.

Reply:
Thank for review. :)

Applying `metav1.GetOptions{ResourceVersion: "0"}` in function directly is for keeping consistent with other use cases.

I have not deep dived into the historic reason why this GET call does not use apiserver cache, but I think, this is a periodic call to check pv/pvc existence and it may not requires strong consistency, so it would be enough to use cache.

Reply:
hi @jfbai, I think either using @mattjmcnaughton's suggestion or adding a comment is necessary here - it is not clear to an observer not intimately familiar with this convention that the behavior of this is to use the cache.

Reply:
@davidz627 Thanks a lot, I will add a brief comment.

Reply:
Is there any documentation available somewhere about how this caching works, and how often it gets updated?